### PR TITLE
增加代扣解约接口

### PR DESCRIPTION
**Pull request by noogel**

### CODE REVIEW

**Comment by messense** [comment]
Please [rebase](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase).

File: wechatpy/pay/api/withhold.py [hunk 1]
```diff
@@ -171,3 +171,28 @@ def query_order(self, transaction_id=None, out_trade_no=None):
...
+        """
+        申请解约
+        :param contract_id: 合同ID
+        :param plan_id: 模板ID
+        :param contract_code: 合同号
+        :param contract_termination_remark: 解约原因
+        :param version: 版本号
+        :return:
+        """
+        if not contract_id and not(plan_id and contract_code):
```
Comment (messense) [inline wechatpy/pay/api/withhold.py]:
```python
if not (contract_id or plan_id or contract_code):
```

File: wechatpy/pay/api/withhold.py [hunk 2]
```diff
@@ -171,3 +171,28 @@ def query_order(self, transaction_id=None, out_trade_no=None):
             "out_trade_no": out_trade_no,
         }
         return self._post("pay/paporderquery", data=data)
+
+    def apply_cancel_signing(self, contract_id=None, plan_id=None, contract_code=None,
+                             contract_termination_remark=None, version="1.0"):
+        """
+        申请解约
```
Comment (messense) [inline wechatpy/pay/api/withhold.py]: 增加以下微信文档链接。

**Review by messense — CHANGES_REQUESTED** [review]

File: wechatpy/pay/api/withhold.py (hunk 2 above)
Comment (noogel) [reply wechatpy/pay/api/withhold.py]: 加上了

File: wechatpy/pay/api/withhold.py (hunk 1 above)
Comment (noogel) [reply wechatpy/pay/api/withhold.py]: 改了 `if not (contract_id or plan_id and contract_code):`

**Review by messense — APPROVED** [review]

**Comment by messense** [comment]
Thanks!

**Pull request merged**